### PR TITLE
Documentation for EKS-A subscriptions

### PR DESCRIPTION
**Pull request by csplinter**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/content/en/docs/clustermgmt/support/cluster-license.md [hunk 1]
```diff
@@ -2,16 +2,79 @@
...
+
+See the steps below for accepting EKS Anywhere license grants with the AWS CLI.
+
+**Get grant ARNs based on subscription `name` with the AWS CLI**
+
+- Replace `region-code` with the AWS Region that hosts your subscription (for example `us-west-2`).
+- Replace `my-subscription` in the `--query` string with the `name` for your subscription.
+
+```bash
+aws license-manager list-received-licenses \
```
Comment (jonathanmeier5) [inline docs/content/en/docs/clustermgmt/support/cluster-license.md:61]: we need to provide the list of license arns to this. the page size of this resposne is only 20 or something. if a customer has a lot of licenses, might not filter correctly.

```
aws license-manager list-received-licenses \
  --license-arns "arn1" "arn2"
  --region 'region-code' \
  --query 'Licenses[?LicenseName==`EKS Anywhere license for subscription my-subscription`].LicenseMetadata[].Value'

File: docs/content/en/docs/concepts/support-scope.md [hunk 2]
```diff
@@ -2,39 +2,83 @@
...
-Bottlerocket OVAs and Raw images are distributed by the EKS Anywhere project.
-See the [Artifacts]({{< relref "../osmgmt/artifacts" >}}) page for details on how to download Bottlerocket images for EKS Anywhere.
+Visit the following links for more information on EKS Anywhere Enterprise Subscriptions
 
-* **Ubuntu**: EKS Anywhere has been tested with Ubuntu-based nodes.
-Amazon will assist with troubleshooting and configuration guidance with Ubuntu-based nodes under the [EKS Anywhere Enterprise Subscription.](https://aws.amazon.com/eks/eks-anywhere/pricing/)
-To build your own Ubuntu-based EKS Anywhere node image, refer to [Building node images]({{< relref "../osmgmt/artifacts/#building-node-images" >}}).
-For official Ubuntu support, see the Canonical [Support](https://ubuntu.com/support) page.
+- [EKS Anywhere Pricing Page](https://aws..com/eks/eks-anywhere/pricing/)
+- [EKS Anywhere FAQ Page](https://aws..com/eks/eks-anywhere/faqs/)
```
Comment (ddjjia) [inline docs/content/en/docs/concepts/support-scope.md:29]:
```suggestion
- [EKS Anywhere Pricing Page](https://aws.amazon.com/eks/eks-anywhere/pricing/)
- [EKS Anywhere FAQ Page](https://aws.amazon.com/eks/eks-anywhere/faqs/)
```

File: docs/content/en/docs/concepts/support-scope.md [hunk 3]
```diff
@@ -2,39 +2,83 @@
...
+If you are using EKS Anywhere and have not purchased a subscription, you can file an [issue](https://github.com/aws/eks-anywhere/issues) in the EKS Anywhere GitHub Repository, and someone will get back to you as soon as possible. If you discover a potential security issue in this project, we ask that you notify AWS/Amazon Security via the [vulnerability reporting page.](http://aws..com/security/vulnerability-reporting/) Please do not create a public GitHub issue for security problems.
 
-## Curated packages support
-Amazon [EKS Anywhere Curated Packages]({{< relref "../packages/" >}}) are Amazon-curated software packages that extend the core functionalities of Kubernetes on your EKS Anywhere clusters.
-All curated packages, including the curated OSS packages, are supported under the EKS Anywhere Enterprise Subscription.
+### FAQs
+
+**1. How much does an EKS Anywhere Enterprise Subscription cost?**
+
+For pricing information, visit the [EKS Anywhere pricing page]((https://aws..com/eks/eks-anywhere/pricing/)).
```
Comment (ddjjia) [inline docs/content/en/docs/concepts/support-scope.md:40]:
```suggestion
For pricing information, visit the [EKS Anywhere pricing page](https://aws.amazon.com/eks/eks-anywhere/pricing/).
```

File: docs/content/en/docs/concepts/support-scope.md [hunk 4]
```diff
@@ -2,39 +2,83 @@
...
+
+**2. How can I purchase an EKS Anywhere Enterprise Subscription?**
+
+Go to the [EKS Anywhere Enterprise Subscription section](https://console.aws.amazon.com/eks/home#/eks-anywhere) of the EKS console. In the console, select your preferred term (1-year or 3-year). Then, specify the license quantity, configure tags and automatic renewal. Review the pending purchase. When the purchase is submitted, you can review licenses associated with your subscription in the console, accept the license grants with AWS License Manager, apply the licenses to your EKS Anywhere clusters, and access EKS Anywhere Curated Packages. For more information, reference the [EKS Anywhere documentation.]({{< relref "../clustermgmt/support/purchase-subscription" >}})
+
+**3. Are subscriptions I previously purchased manually integrated into the  EKS console?**
```
Comment (ddjjia) [inline docs/content/en/docs/concepts/support-scope.md:46]:
```suggestion
**3. Are subscriptions I previously purchased manually integrated into the EKS console?**
```

File: docs/content/en/docs/concepts/support-scope.md [hunk 5]
```diff
@@ -2,39 +2,83 @@
...
+
+No, EKS Anywhere Enterprise Subscriptions purchased manually before October 2023 cannot be viewed or managed through the  EKS console, APIs, and AWS CLI.
```
Comment (ddjjia) [inline docs/content/en/docs/concepts/support-scope.md:48]:
```suggestion
No, EKS Anywhere Enterprise Subscriptions purchased manually before October 2023 cannot be viewed or managed through the EKS console, APIs, and AWS CLI. 
```

File: docs/content/en/docs/clustermgmt/support/cluster-license.md (hunk 1 above)
Comment (csplinter) [reply docs/content/en/docs/clustermgmt/support/cluster-license.md:72]: I added a `--filter 'Name=IssuerName,Values=Amazon EKS Anywhere'` ... that should be sufficient for most use cases

File: docs/content/en/docs/concepts/support-scope.md [hunk 6]
```diff
@@ -2,39 +2,83 @@
...
-All curated packages, including the curated OSS packages, are supported under the EKS Anywhere Enterprise Subscription.
+### FAQs
+
+**1. How much does an EKS Anywhere Enterprise Subscription cost?**
+
+For pricing information, visit the [EKS Anywhere pricing page](https://aws.amazon.com/eks/eks-anywhere/pricing/).
+
+**2. How can I purchase an EKS Anywhere Enterprise Subscription?**
+
+Go to the [EKS Anywhere Enterprise Subscription section](https://console.aws.amazon.com/eks/home#/eks-anywhere) of the EKS console. In the console, select your preferred term (1-year or 3-year). Then, specify the license quantity, configure tags and automatic renewal. Review the pending purchase. When the purchase is submitted, you can review licenses associated with your subscription in the console, accept the license grants with AWS License Manager, apply the licenses to your EKS Anywhere clusters, and access EKS Anywhere Curated Packages. For more information, reference the [EKS Anywhere documentation.]({{< relref "../clustermgmt/support/purchase-subscription" >}})
```
Comment (chrisnegus) [inline docs/content/en/docs/concepts/support-scope.md:44]:
```suggestion
Go to the [EKS Anywhere Enterprise Subscription section](https://console.aws.amazon.com/eks/home#/eks-anywhere) of the EKS console. In the console, select your preferred term (1-year or 3-year). Then, specify the license quantity, configure tags, and choose automatic renewal (optional). Review the pending purchase. When the purchase is submitted, you can review licenses associated with your subscription in the console, accept the license grants with AWS License Manager, apply the licenses to your EKS Anywhere clusters, and access EKS Anywhere Curated Packages. For more information, reference the [EKS Anywhere documentation.]({{< relref "../clustermgmt/support/purchase-subscription" >}})
```
This is a pretty specific list for an FAQ section. If this information is elsewhere, you might want to point there and not risk this text going out of date.

File: docs/content/en/docs/concepts/support-scope.md [hunk 7]
```diff
@@ -2,39 +2,83 @@
...
+
+No, EKS Anywhere Enterprise Subscriptions purchased manually before October 2023 cannot be viewed or managed through the EKS console, APIs, and AWS CLI. 
+
+**4. Can I cancel my subscription in the EKS console, APIs, and AWS CLI?**
+
+You can cancel your subscription within the first 7 days of purchase by filing an AWS Support ticket. When you cancel your subscription within the first 7 days, you are not charged for the subscription. To cancel your subscription outside of the 7-day time period, contact your AWS account team.
+
+**5. In which AWS Regions can I purchase subscriptions?**
+
+You can purchase subscriptions in US East (Ohio), US East (N. Virginia), US West (N. California), US West (Oregon), Africa (Cape Town), Asia Pacific (Hong Kong), Asia Pacific (Jakarta), Asia Pacific (Mumbai), Asia Pacific (Osaka), Asia Pacific (Seoul), Asia Pacific (Singapore), Asia Pacific (Sydney), Asia Pacific (Tokyo), Canada (Central), Europe (Frankfurt), Europe (Ireland), Europe (London), Europe (Milan), Europe (Paris), Europe (Stockholm), Middle East (Bahrain), South America (Sao Paulo).
```
Comment (chrisnegus) [inline docs/content/en/docs/concepts/support-scope.md:56]:
```suggestion
You can purchase subscriptions in US East (Ohio), US East (N. Virginia), US West (N. California), US West (Oregon), Africa (Cape Town), Asia Pacific (Hong Kong), Asia Pacific (Jakarta), Asia Pacific (Mumbai), Asia Pacific (Osaka), Asia Pacific (Seoul), Asia Pacific (Singapore), Asia Pacific (Sydney), Asia Pacific (Tokyo), Canada (Central), Europe (Frankfurt), Europe (Ireland), Europe (London), Europe (Milan), Europe (Paris), Europe (Stockholm), Middle East (Bahrain), and South America (Sao Paulo).
```

File: docs/content/en/docs/concepts/support-scope.md [hunk 8]
```diff
@@ -2,39 +2,83 @@
...
+
+**6. Can I renew my subscription through the EKS console, APIs, and AWS CLI?**
+
+Yes, you can configure auto renewal during subscription creation or at any time during your subscription term. When auto renewal is enabled for your subscription, the subscription and associated licenses will be automatically renewed for the term of the existing subscription (1-year or 3-years). The 7-day cancellation period does not apply to renewals. You do not need to reapply licenses to your EKS Anywhere clusters when subscriptions are automatically renewed.
+
+**7. Can I edit my subscription through the EKS console, APIs, and AWS CLI?**
+
+You can edit the auto renewal and tags configurations for your subscription with the  EKS console, APIs, and AWS CLI. To change the term or license quantity for a subscription, you must create a new subscription.
```
Comment (chrisnegus) [inline docs/content/en/docs/concepts/support-scope.md:64]:
```suggestion
You can edit the auto renewal and tags configurations for your subscription with the EKS console, APIs, and AWS CLI. To change the term or license quantity for a subscription, you must create a new subscription.
```

File: docs/content/en/docs/concepts/support-scope.md [hunk 9]
```diff
@@ -2,39 +2,83 @@
...
+
+**8. What happens when a subscription expires?**
+
+When subscriptions expire, licenses associated with the subscription can no longer be used for new support tickets, access to EKS Anywhere Curated Packages is revoked, and you are no longer billed for the subscription. Support tickets created during the active subscription period will continue to be serviced. You will receive emails 3 months, 1 month, and 1 week before subscriptions expire, and an alert is presented in the EKS console for approaching expiration dates. Subscriptions can be viewed with the EKS console, APIs, and AWS CLI after expiration.
+
+**9. Can I share access to curated packages with other AWS accounts?**
+
+Yes, reference the [documentation]({{< relref "../clustermgmt/support/share-packages" >}}) for instructions on how to share access to curated packages with other AWS accounts in your organization.
```
Comment (chrisnegus) [inline docs/content/en/docs/concepts/support-scope.md:72]:
```suggestion
Yes, reference [Share curated packages access]({{< relref "../clustermgmt/support/share-packages" >}}) documentation for instructions on how to share access to curated packages with other AWS accounts in your organization.
```

File: docs/content/en/docs/concepts/support-scope.md [hunk 10]
```diff
@@ -2,39 +2,83 @@
...
+
+**10. How do I apply licenses to my EKS Anywhere clusters?**
+
+Reference the [documentation]({{< relref "../clustermgmt/support/cluster-license" >}}) for instructions on how to apply licenses your EKS Anywhere clusters.
```
Comment (chrisnegus) [inline docs/content/en/docs/concepts/support-scope.md:76]:
```suggestion
Reference the [License cluster]({{< relref "../clustermgmt/support/cluster-license" >}}) documentation for instructions on how to apply licenses your EKS Anywhere clusters.
```

**Review by chrisnegus — COMMENTED** [review]
A few editing suggestions.

File: docs/content/en/docs/clustermgmt/support/purchase-subscription.md [hunk 11]
```diff
@@ -0,0 +1,204 @@
...
+1. Open the Amazon EKS console at https://console.aws.amazon.com/eks/home#/eks-anywhere
+1. Click the **Create subscription** button on the right side of the screen.
+1. On the **Specify subscription details** page, select an offer (1 year term or 3 year term)
+1. Configure the following fields
+
+  - **Name** - a name for your subscription. It must be unique in your AWS account in the AWS Region you're creating the subscription in. The name can contain only alphanumeric characters (case-sensitive), hyphens, and underscores. It must start with an alphabetic character and can't be longer than 100 characters. This value cannot be changed after creating the subscription.
+  - **Number of licenses** - the number of licenses to include in the subscription. This value cannot be changed after creating the subscription.
+  - **Auto renewal** - if enabled, the subscription will automatically renew at the end of the term.
+
+5. (Optional) **Configure tags**. A tag is a label that you assign to an EKS Anywhere subscription. Each tag consists of a key and an optional value. You can use tags to search and filter your resources.
```
[the remark this answers is not in the document]
Comment (csplinter) [reply docs/content/en/docs/clustermgmt/support/purchase-subscription.md:37]: You can have a key without a value, just tested this in the UI

File: docs/content/en/docs/concepts/support-scope.md (hunk 6 above)
Comment (csplinter) [reply docs/content/en/docs/concepts/support-scope.md]: ack, linked out to the docs instead

**Pull request merged**